server/container/cli/src/main/java/org/apache/james/cli/ServerCmd.java

```diff
@@ -94,7 +94,7 @@ public static void main(String[] args) {
         } catch (IOException ioe) {
             failWithMessage("Error connecting to remote JMX agent : " + ioe.getMessage());
         } catch (Exception e) {
-            LOG.error("Error on command: {}", e);
+            LOG.error("Error on command: {}", e.toString());
```

Comment:
Swallowing stacktrace ;-)

I would be in favour of:

```
LOG.error("Error while playing command", e);
```

Reply:
Done